pymc3/plots.py

```diff
@@ -84,7 +84,7 @@ def histplot_op(ax, data):
         mind = np.min(d)
         maxd = np.max(d)
         step = max((maxd-mind)//100, 1)
-        ax.hist(d, bins=range(mind, maxd + 2, step), align='left')
+        ax.hist(d, bins=range(mind, maxd + 2, step), alpha=.3, align='left')
```

Comment:
Yeah, makes sense the user should be allowed to do that. Not quite sure how to pass it around since there are three different kinds of plots. Should they all use the same alpha? Should we have three sets of dicts that they can pass that get passed to those plots?

Thoughts?